src/librbd/AsyncResizeRequest.cc

```diff
@@ -317,4 +316,11 @@ void AsyncResizeRequest::update_size_and_overlap() {
   }
 }
 
+void AsyncResizeRequest::send_notify_change() {
+  ldout(m_image_ctx.cct, 5) << this << " send_grow_object_map: "
+                            << " original_size=" << m_original_size
+                            << " new_size=" << m_new_size << dendl;
+  notify_change(m_image_ctx.md_ctx, m_image_ctx.header_oid, &m_image_ctx);
```

Comment:
```notify_change``` isn't an async operation -- it will potentially block until the notify timeout hits.